engine/src/main/scala/cromwell/engine/workflow/ShadowWorkflowActor.scala

```diff
@@ -96,7 +96,7 @@ object ShadowWorkflowActor {
 class ShadowWorkflowActor(workflowId: WorkflowId,
                           startMode: StartMode,
                           workflowSources: WorkflowSourceFiles,
-                          conf: Config) extends LoggingFSM[ShadowWorkflowActorState, ShadowWorkflowActorData] with ActorLogging{
+                          conf: Config) extends LoggingFSM[ShadowWorkflowActorState, ShadowWorkflowActorData] with ActorLogging {
```

Comment:
I'm the reviewer and it's my humble opinion. :)

When git barfs with a merge conflict, I'm generally searching through the conflicting lines for "why did someone touch this line?" I personally wouldn't expect someone just to add a single space in the middle of 30+ other files. This line _will_ change eventually, and should be fixed then.

Similarly, when I do git blame, and the line is marked as "[cjllanwarne](https://github.com/broadinstitute/cromwell/blame/e11ca390495247eea2f3c46409faeb47a3ab5f25/engine/src/main/scala/cromwell/engine/workflow/ShadowWorkflowActor.scala#L99)" I don't end up chasing down "[scottfrazer](https://github.com/broadinstitute/cromwell/blame/8b8f7ed10d41a66f3afd8a6dbb6a8500b241d4c3/engine/src/main/scala/cromwell/engine/workflow/ShadowWorkflowActor.scala#L99)" instead.

Lastly, another way of putting it-- if one is fixing this one whitespace character in this one file, why doesn't this PR also fix other spaces in other files? Usually when one is on a cleanup hunt, the entire commit is labeled as such.